README.rst

```diff
@@ -188,7 +187,50 @@ If you prefer to use NeMo's latest development version (from GitHub) follow the
     python setup.py style --fix  # Tries to fix error in-place.
     python setup.py style --scope=tests  # Operates within certain scope (dir of file).
 
-** NeMo Test Suite**
+
+** NeMo Documentation **
+
+.. |master| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=master
+  :alt: Documentation Status
+  :scale: 100%
+  :target: https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/master/
+
+.. |latest| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=latest
+  :alt: Documentation Status
+  :scale: 100%
+  :target: https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/latest/
+
+.. |stable| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=stable
+  :alt: Documentation Status
+  :scale: 100%
+  :target: https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/stable/
+
+.. |v0110| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=v0.11.0
+  :alt: Documentation Status
+  :scale: 100%
+  :target: https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/v0.11.0/
+
+.. |v0101| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=v0.10.1
+  :alt: Documentation Status
+  :scale: 100%
+  :target: https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/v0.10.1/
+
+
++---------+----------+---------------------------------------------------------+
+| Version | Status   | Description                                             |
++=========+==========+=========================================================+
+| Master  | |master| | Documentation of the `master` branch                    |
++---------+----------+---------------------------------------------------------+
+| Latest  | |latest| | Documentation of the latest (i.e. `master`) branch      |
++---------+----------+---------------------------------------------------------+
```

Comment:
Do we need both of these?

Reply:
You tell me - the table only reflects the links that are available in version RTD selector.

 I raised that subject long time ago, heard back then that we want `master`...